website/docs/r/portal_tenant_configuration.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+subcategory: "Portal"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_portal_tenant_configuration"
+description: |-
+  Manages Portal Tenant Configuration.
+---
+
+# azurerm_portal_tenant_configuration
+
+Manages Portal Tenant Configuration.
+
+~> **Note:** User has to be a Tenant Admin for managing this resource. The Service Principal with Tenant Admin could be created by `az ad sp create-for-rbac --name "<sp name>" --role="Owner" --scopes="/"`.
+
+## Example Usage
+
+```hcl
+resource "azurerm_portal_tenant_configuration" "example" {
+  enforce_private_markdown_storage = true
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `enforce_private_markdown_storage` - (Required) Is Markdown tile which used to display custom and static content enabled?
```

Comment:
the property name doesn't match the description?
```suggestion
* `enforce_private_markdown_storage` - (Required) Is the private tile markdown storage which used to display custom dynamic and static content enabled?
```

Reply:
Updated